glyph-brush/src/glyph_brush.rs

```diff
@@ -417,7 +417,7 @@ struct LastDrawInfo {
 // (screen_width, screen_height): (f32, f32),
 
 /// Data used to generate vertex information for a single glyph
-#[derive(Debug)]
+#[derive(Debug, Copy, Clone, PartialEq)]
```

Comment:
I'm not sure about making `GlyphVertex` more convenient in this way as I want to encourage users to convert this in the closure & not save it. What's the use case for these implementations?